src/memefactory/ui/get_dank/events.cljs

```diff
@@ -28,6 +29,44 @@
    {:db (assoc db
           ::spinner false)}))
 
+(re-frame/reg-event-fx
+ ::stage
+ (fn [{:keys [db]} [_ stage-number]]
+   {:db (assoc db :memefactory.ui.get-dank.page/stage stage-number)}))
+
+(defn get-allocated-dank [db phone-number]
```

Comment:
So this function is not used now? If not, please remove

Reply:
@madvas My bad, missed that. Removed the unused fn.